public/openapi/v1.0-pre.yaml

```diff
@@ -2699,20 +2672,16 @@ paths:
                       type: string
                     course_subject_three:
                       type: string
-                    course_max_age:
-                      type: integer
-                    lead_school_urn:
-                      type: string
                     employing_school_urn:
                       type: string
                     trn:
                       type: string
+                    lead_partner_urn:
+                      type: string
                     ethnic_group:
                       type: string
                     ethnic_background:
                       type: string
```

Comment:
Looks like we still have `ethnic_group` and `ethnic_background` in this request body schema, which I think we want to remove.